Makefile

```
@@ -9,15 +9,35 @@ else
   CIBRANCH=$(shell git rev-parse --abbrev-ref HEAD | sed 's/\//-/')
 endif
 
+ifndef RELEASE_ARCH
+override RELEASE_ARCH=amd64
+endif
+
+ifndef LIBC
+override LIBC=musl
+endif
```

My gut tells me these shouldn't be in the Makefile at all. But if they must be, `?=` is the idiomatic way of conditionally setting variables.
```suggestion
RELEASE_ARCH?=amd64
LIBC?=musl
```

---

that switch was debugging build issues that turned out to be a mismatch between the host and the container during build time but I left them b/c supposedly the `override` syntax will handle when the value is defined but `null` whereas `?=` will only handle when the value isn't yet defined? in any case, i'm going to try and strip those out and default the target to "whatever you're building for" but i need to figure out what that behavior looks like inside a production build alpine container and make sure i can detect and override that if needed.